src/core.js

```diff
@@ -436,6 +436,18 @@
     // Build the upstream conduit communicating with the CCP iframe.
     var conduit = new connect.IFrameConduit(params.ccpUrl, window, iframe);
 
+    // Let CCP know if iframe is visible
+    iframe.onload = setTimeout(function() {
+      var style = window.getComputedStyle(iframe, null);
+      var data = {
+        display: style.display,
+        offsetWidth: iframe.offsetWidth,
+        offsetHeight: iframe.offsetHeight,
+        clientRectsLength: iframe.getClientRects().length
+      };
+      conduit.sendUpstream(connect.EventType.IFRAME_STYLE, data);
```

Comment:
Does this `EventType` exist?

Reply:
hmm I added it in event.js but I guess it didn't get committed. I tested and made sure it worked / i'll add it back in